src/components/tasks/card/SuggestionBox.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useEffect, useState } from 'react';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+import classes from '@/components/tasks/card/suggestion.module.scss';
+import fetch from '@/helperFunctions/fetch';
+import { userDataType } from '@/interfaces/user.type';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+
+type Props = {
+    assigneeName: string;
+    showSuggestion: boolean;
+    onClickName: (userName: string) => void;
+};
```

Comment:
please remove these also in suggestionBox.type

Reply:
I do not think we need to put prop types in different file? Props are localized, they belong to particular component.

Reply:
follow either of the conventions check throughout the project and see how the types are actually structured if all types are taken out into a types file do it else if local types are kept local follow that.